source/software/r_package_management.rst

```diff
@@ -105,17 +88,19 @@ If the result is blank, or reports that conda can not be found, modify
 the \`PATH\` environment variable appropriately by adding miniconda's bin
 directory to PATH.
 
-Creating a new conda environment is straightforward::
+The next step is to create a new conda environment which can be done as follows::
+
+   $ conda search -c conda-forge r-base  # select one of available versions for the step below
+   $ conda create -n science -c conda-forge r-base=<version> r-essentials
+   
 
-   $ conda create -n science -c r r-essentials r-rodbc
+This command creates a new conda environment called "science", and installs your prefered R 
+version from the conda-forge channel as well as the r-essentials bundle which includes number
+of commonly used R packages such as ggplot2, glmnet, dplyr, tidyr, and shiny.
```

Comment:
following our style for the vsc pages, please change the typesetting for the package names by sandwiching them with opening/closing double-backticks; so that instead of ggplot2, we eventually render `ggplot2`.